src/main/resources/api/openapi.yaml

```diff
@@ -1030,3 +1076,140 @@ paths:
+    get:
+      summary: Get total amount of confirmed Ergo tokens and assets
+      operationId: walletBalances
+      tags:
+        - wallet
+      responses:
+        '200':
+          description: Get total amount of confirmed Ergo tokens and assets
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BalancesSnapshot'
+        default:
+          description: Error
+          content:
+            text/plain; charset=UTF-8:
+              schema:
+                type: string
+
+
+  /wallet/balances/unconfirmed:
+    get:
+      summary: Get amount of unconfirmed Ergo tokens and assets
+      operationId: walletBalancesUnconfirmed
+      tags:
+        - wallet
+      responses:
+        '200':
+          description: Get amount of unconfirmed Ergo tokens and assets
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BalancesSnapshot'
+        default:
+          description: Error
+          content:
+            text/plain; charset=UTF-8:
+              schema:
+                type: string
+
+  /wallet/transaction:
+    post:
+      summary: Send a transaction
+      operationId: walletTransactionSend
+      tags:
+      - wallet
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ErgoTransaction'
+      responses:
+        '200':
+          description: JSON with ID of the new transaction
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TransactionId'
+        default:
+          description: Error
+          content:
+            text/plain; charset=UTF-8:
+              schema:
+                type: string
+                example: invalid.transaction.signature, invalid.transaction.timestamp, invalid.transaction.value, invalid.box.nonce, invalid.json
+
+
+  /wallet/transaction/generate:
+    post:
+      summary: Generate a transaction from payment
+      operationId: walletTransactionGenerate
+      tags:
+        - wallet
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/PaymentRequest'
+      responses:
+        '200':
+          description: Generated Ergo transaction
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErgoTransaction'
+        '400':
+          description: Bad payment request
+          content:
+            text/plain; charset=UTF-8:
+              schema:
+                type: string
+        default:
+          description: Error
+          content:
+            text/plain; charset=UTF-8:
+              schema:
+                type: string
+
+  /wallet/transaction/payment:
```

Comment:
Example request
```
[
  {
    "address": "string",
    "value": 0,
    "assets": [
      {
        "tokenId": "4ab9da11fc216660e974842cc3b7705e62ebb9e0bf5ff78e53f9cd40abadd117",
        "amount": 1000
      }
    ],
    "registers": {
      "R4": "7c"
    }
  }
]
```
does not work here with correct amount and address

Reply:
this was because of register value (earlier it worked), fixed